### ISSUE TITLE
IFC Search ignoring case

### ISSUE THREAD

**Issue by JanFilipec** [IfcOpenShell/IfcOpenShell#1339]
This is a tiny thing, but cetainly confusing-the search ignores case of the searched term, but not of the Attribute so:

Object: Name=Wall

Search: Name=wall finds
Seach: NAME=Wall doesn't

**Comment by Moult** [comment]
Hmm yeah - that is how the "Ignore case" option is meant to work - it ignores the case of your search keyword, not the filter... I'm not sure quite how I feel about this. Attributes are case sensitive I believe, so are property set names and property names, so making it case insensitive can create issues (though admittedly I've never seen such a scenario in practice).

Thoughts, @aothms ?

**Comment by JanFilipec** [comment]
I'm not quite sure what you mean, is the search algorithm used for something else than the mentioned search function? That it can theoretically lead to elements with different properties differing only in case being selected is hopefully pretty obvious? But being forced to remember the correct capitalization of the properties seems quite exhausting to me.
Anyway for now I would perhaps just suggest rewording the option to "ignore case of the searched term" instead of "search ignoring case".

**Comment by aothms** [comment]
< Attributes are case sensitive I believe
